nixos/tests/prowlarr.nix

```diff
@@ -1,22 +1,36 @@
-import ./make-test-python.nix (
-  { lib, ... }:
+{
+  runTest,
+  dataDir,
+}:
 
-  {
-    name = "prowlarr";
-    meta.maintainers = with lib.maintainers; [ ];
+runTest
 
-    nodes.machine =
-      { pkgs, ... }:
-      {
-        services.prowlarr.enable = true;
-      };
+  (
+    { lib, ... }:
 
-    testScript = ''
-      machine.wait_for_unit("prowlarr.service")
-      machine.wait_for_open_port(9696)
-      response = machine.succeed("curl --fail http://localhost:9696/")
-      assert '<title>Prowlarr</title>' in response, "Login page didn't load successfully"
-      machine.succeed("[ -d /var/lib/prowlarr ]")
-    '';
-  }
-)
+    {
+      name = "prowlarr";
+      meta.maintainers = with lib.maintainers; [ ];
+
+      nodes.machine =
+        { pkgs, ... }:
+        {
+          services.prowlarr = {
+            enable = true;
+            inherit dataDir;
+          };
+        };
+
+      testScript = ''
+        machine.wait_for_unit("prowlarr.service")
+        machine.wait_for_open_port(9696)
+        response = machine.succeed("curl --fail http://localhost:9696/")
+        assert '<title>Prowlarr</title>' in response, "Login page didn't load successfully"
+        machine.succeed("[ -d ${dataDir} ]")
+        # Check that we're using a bind mount when using a non-default dataDir
```

Comment:
IMO this is an implementation detail, not worth testing. I'll approve the PR because this is somewhat nitpicky.